deployment/deployment-pipelines.yml

```diff
@@ -7,16 +26,45 @@ jobs:
     matrix:
       Python36:
         PYTHON_VERSION: '3.6'
+  variables:
+    valid_version: 'false'
 
   steps:
   - script: |
+      echo $(build_version)
+      if [[ $(build_version) =~ v[0-9](.[0-9])+ ]]; then
```

Comment:
suggest to limit the number of .[0-9], like:
v[0-9](.[0-9]){1,3} is enough

Reply:
Thanks, updated.